server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
```

Comment:
when you just have a type for a field you can write it like so `points: String`. not critical but might make everything a bit easier to read